rendezvous-server/src/handlers_to0.rs

```diff
@@ -157,8 +136,15 @@ pub(super) async fn ownersign(
         }
         Some(v) => v,
     };
-    //let device_pubkey = match device_cert_chain.verify_from_x5bag(&user_data.trusted_device_keys) {
-    let device_pubkey = match device_cert_chain.insecure_verify_without_root_verification() {
+
+    let device_pubkey_verification =
+        if let Some(trusted_manufacturer_certs) = &user_data.trusted_manufacturer_keys {
+            device_cert_chain.verify_from_x5bag(trusted_manufacturer_certs)
```

Comment:
So, I think that the issue here is whether we can trust any certificate as root or not and whether the rendezvous has the abilities to verify root certificates...

Reply:
what was the error that we were getting before changing the ca cert in the tests?

Reply:
> what was the error that we were getting before changing the ca cert in the tests?

Honestly, I don't remember, I would need to check the conformance tests again. 
Either way, we need the servers to be able to validate the device certificate chain by using a list of trusted CA certs (apart from verifying that the chain is correct).